tests/integration/test_github.py

```diff
@@ -332,3 +332,11 @@ def test_create_missing_coverage_annotation__annotation_type(capsys):
     assert (
         output.out.strip() == "::error file=test.py,line=42::This line has no coverage"
     )
+
+
+def test_add_job_summary(summary_file):
+    github.add_job_summary(
+        content="[job summary content]", github_step_summary=summary_file
+    )
```

Comment:
Can we call it twice to check that content was appended and not truncated ?

Reply:
Yes, good idea